packages/puppeteer-extra-plugin-stealth/evasions/navigator.hardwareConcurrency/readme.md

```diff
@@ -0,0 +1,24 @@
+## API
+
+<!-- Generated by documentation.js. Update this documentation by updating the source code. -->
+
+#### Table of Contents
+
+- [class: Plugin](#class-plugin)
+
+### class: [Plugin](https://github.com/berstend/puppeteer-extra/blob/e6133619b051febed630ada35241664eba59b9fa/packages/puppeteer-extra-plugin-stealth/evasions/navigator.webdriver/index.js#L9-L23)
+
+- `opts` (optional, default `{}`)
+
+**Extends: PuppeteerExtraPlugin**
+
+Allows faking CPU core number. This allows creating different profiles.
+Will overwrite `navigator.hardwareConcurrency` property.
+
+####Usage:
```

Comment:
It doesn't seem like `Generated by documentation.js.` is true here 😄 Please add the text to the JSDoc comment in the code and run `yarn docs-for-evasions` in the `puppeteer-extra/packages/puppeteer-extra-plugin-stealth`folder to auto-generate the markdown file based on the code comments.

Reply:
oops. I have truncated the readme. Added comments. but yarn won't run on my windows system.

Reply:
Would be good if you find a way (WSL or something), I also noticed that tests are missing which are required for evasions (otherwise the code wouldn't be maintainable anymore). :-)

Reply:
Thanks a lot for all this. I feel being mentored :)
hmm yarn works on WSL. but fails with 'loop: not found'. I believe its failing at 
`"postdocs-for-evasions": "cd ./evasions && loop \"npx prettier --write readme.md\""`

I am not really great with nodejs. writing test would be hard and could be prone to error ..